### PR TITLE
Added AALIYA as contributor in CONTRIBUTOR.md

### PR DESCRIPTION
**Pull request by Aaliya7516**
Concerns with issue #8 
@pixan198 please review

### CODE REVIEW

File: CONTRIBUTORS.md [hunk 1]
```diff
@@ -80,6 +80,20 @@ Deepak suryawanshi
 &#47;&gt;</a>
 </td>
 
+<td>
+<a href="https://www.linkedin.com/in/aaliya7516" >
+<img src="https://avatars1.githubusercontent.com/u/53328555?v=4" alt="Aaliya Haque" title="Aaliya Haque" width="100px" height="auto" />
```
Comment (ashish11chawda) [inline CONTRIBUTORS.md:85]: Can you please change the image url to your GitHub avatar url?

Comment (Aaliya7516) [reply CONTRIBUTORS.md:85]: yeah sure

**Comment by Aaliya7516** [comment]
changed the image url

**Comment by ashish11chawda** [comment]
Looks Good 👍

**Pull request merged**